_posts/2016-12-04-contributing-blog-post-to-pulpproject.org.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Contributing a blog post to pulpproject.org
+layout: post
+author: Dennis Kliban
+---
+1. Fork the pulpproject.org repository on GitHub.
+   [https://github.com/pulp/pulpproject.org/](https://github.com/pulp/pulpproject.org/)
+
+1. Clone your fork locally and checkout the `gh-pages` branch.
+
+1. Create a new file in the `_posts` directory. The name for the file should be in the following
+   format: `2016-11-30-contributing-blog-post-to-pulpproject.org.md`
+
+1. Write a blog post in the new file.
+
+   Pulpproject.org is a static site generated from
+   [Markdown](https://daringfireball.net/projects/markdown/) documents using
+   [Jekyll](http://jekyllrb.com/). Each blog post begins with the [YAML Front
+   Matter](http://jekyllrb.com/docs/frontmatter/). Front Matter begins and ends with `===`. In
+   between the dashes, the autor species a `layout`, an `author`, a `title`, and `tags`. All blog
```

Comment:
s/autor/author/
Also the tags are optional so that should be its own sentence probably.